extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/devconsole/ContainerImageDevUiProcessor.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.container.image.deployment.devconsole;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import io.quarkus.bootstrap.BootstrapException;
+import io.quarkus.bootstrap.app.ArtifactResult;
+import io.quarkus.bootstrap.app.AugmentResult;
+import io.quarkus.bootstrap.app.CuratedApplication;
+import io.quarkus.bootstrap.app.QuarkusBootstrap;
+import io.quarkus.container.image.runtime.devui.ContainerBuilderJsonRpcService;
+import io.quarkus.container.spi.AvailableContainerImageExtensionBuildItem;
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.dev.console.DevConsoleManager;
+import io.quarkus.dev.console.TempSystemProperties;
+import io.quarkus.devui.spi.JsonRPCProvidersBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.vertx.core.json.JsonArray;
+
+public class ContainerImageDevUiProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create(List<AvailableContainerImageExtensionBuildItem> extensions) {
+        // Get the list of builders
+        JsonArray array = extensions.stream().map(AvailableContainerImageExtensionBuildItem::getName).sorted()
+                .collect(JsonArray::new, JsonArray::add, JsonArray::addAll);
+
+        CardPageBuildItem card = new CardPageBuildItem("Container Image");
+        card.addBuildTimeData("builderTypes", array);
+        card.addPage(Page.webComponentPageBuilder()
+                .title("Build Container")
+                .componentLink("build-container-component.js")
```

Comment:
component should follow the conversion qwc-extensionName-pageName so `qwc-container-image-build`